enable/qt4/quartz.py

```diff
@@ -7,68 +7,22 @@
 # is also available online at http://www.enthought.com/licenses/BSD.txt
 #
 # Thanks for using Enthought open source!
-import numpy as np
+from PIL import Image, ImageQt
+
+# Qt imports.
+from pyface.qt import QtCore, QtGui
 
 from kiva.api import Font
-from kiva.quartz import get_mac_context, ABCGI
+from kiva.quartz import ABCGI
```

Comment:
Now that `get_mac_context` is not needed, we should revisit whether to include it.